bin/hdfcoinc/pycbc_fit_sngls_by_template

```diff
@@ -122,6 +122,20 @@ trigf = h5py.File(args.trigger_file, 'r')
 logging.info('Opening template file: %s' % args.bank_file)
 templatef = h5py.File(args.bank_file, 'r')
 
+# Count the number of triggers in each template, for calulation of the true noise rate
+logging.info('Counting number of triggers in each template')
+tb = trigf[args.ifo+'/template_boundaries'][:]
+hash_sort = np.argsort(templatef['template_hash'][:])
+tb_sorted = tb[hash_sort]
+# calculate the differences between the boundary indices, 
+# in order to get the number in each template
+count_in_template = tb_sorted[1:] - tb_sorted[:-1]
+#Add on the last template
+count_in_template = np.append(count_in_template,
+                               [len(trigf[args.ifo+'/template_id'][:]) - tb_sorted[-1]])
+
+tid_sort = np.argsort(hash_sort)
```

Comment:
this is confusing to me - hash sort is already an argsort, from line 128, what is being done here?

Reply:
by sorting the hash_sort we get an order in template id (the original order) which will put count_in_template back into the order of the original list

Reply:
so for any list b:
`all(b[np.argsort(b)][np.argsort(np.argsort(b))]==b)` will be True

Reply:
ok, if you think this is the simplest way to achieve that can you write a comment to explain it ..